src/Cpu.re

```diff
@@ -9,6 +9,18 @@ type cpu = {
   mutable pc: int,
 };
 
+type cpu = t;
```

Comment:
I'm a little confused, is this still being used anywhere? Seems we could maybe get rid of the `cpu` type alias here.